src/StatCache.cc

```diff
@@ -59,7 +59,7 @@ int StatCache::stat(const char *path, struct stat *st)
 		}
 		statMap.erase(iter);                            // Remove stale entry
 	}
-	CacheEntry entry{};                               // Make a new entry
+	CacheEntry entry;                                 // Make a new entry
```

Comment:
Both mingw64 in the msys2 on Windows 64 build and whatever Travis uses for the Linux builds. 

I don't know C++11 because I left my job writing C++ in 2011 to make Mendel90 kits! I used to initialise structs with = {0} but that gives exactly the same warnings with these compilers.